frontend/packages/pipelines-plugin/src/components/repository/RepositoryLinkList.tsx

```diff
@@ -0,0 +1,62 @@
+import * as React from 'react';
+import { GithubIcon } from '@patternfly/react-icons';
+import { useTranslation } from 'react-i18next';
```

Comment:
nit: sort imports

Reply:
I started off as that but it  gives a lint error
![Screenshot from 2021-08-03 14-00-14](https://user-images.githubusercontent.com/24852534/127987696-49af3944-ce85-4598-b103-0f5de78e8269.png)